flutter-examples/non_streaming_vad_asr/README.md

```diff
@@ -0,0 +1,53 @@
+# Real-time speech recognition
```

Comment:
PLEASE update this file.

Make sure that users can follow this file to run the demo.